message_sender/views.py

```diff
@@ -109,12 +109,27 @@ def post(self, request, *args, **kwargs):
                         message.metadata["delivery_timestamp"] = \
                             request.data["timestamp"]
                         message.save()
+
+                        # OBD number of successful tries metric
+                        if "voice_speech_url" in message.metadata:
+                            fire_metric.apply_async(kwargs={
+                                "metric_name":
+                                    'vumimessage.obd.successful.sum',
+                                "metric_value": 1.0
+                            })
```

Comment:
This is a bit of a difficult one, because we don't always get delivery reports for all transports. I think placing this under the `ack` (acknowledge) section (which indicates that we handed it over properly), will give us more accurate metrics, since we are never guaranteed delivery reports (and we're also not guaranteed only a single delivery report, it's possible to get duplicates), but we are guaranteed a single ack/nack for each message.